07.bowling_object/src/main.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require_relative './game'
+
+def main
+  game = Game.new(ARGV.shift)
+  puts game.calc_score
+end
+
+main
```

Comment:
他にメソッドがなく、このファイル自体が`main`と付けられているので、

```suggestion
game = Game.new(ARGV.shift)
puts game.calc_score
```

だけでも良いかもしれません

Reply:
修正しました！